hardware/system/upboard-4000/install_script.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+set -e
+
+MACHINE=$1
+TAG=$2
+PRINTER=$3
+
+if [ -z $1 ] || [ -z $2 ]; then
+  echo 'usage: install_script <machine> <version> [printer]'
+  echo 'machines: "gaia", "sintra" or "tejo"'
+  echo 'version: git tag'
+  echo 'printer [optional]: "nippon" or "zebra"'
+  exit 1
+fi
+
+if [ "$MACHINE" != "gaia" ] && [ "$MACHINE" != "sintra" ] && [ "$MACHINE" != "tejo" ]; then
+  echo 'Install script expects "gaia", "sintra" or "tejo" as machine parameter'
+  exit 1
+fi
+
+if [ -z $3 ]; then
+  PRINTER='None'
+fi
+
+if [ "$PRINTER" != "None" ] && [ "$PRINTER" != "zebra" ] && [ "$PRINTER" != "nippon" ]; then
+  echo 'Install script expects "zebra" or "nippon" as printer parameter'
+  exit 1
+fi
+
+if [ "$PRINTER" == "nippon" ]; then
+  PRINTER='Nippon-2511D-2'
+fi
+
+if [ "$PRINTER" == "zebra" ]; then
+  PRINTER='Zebra-KR-403'
+fi
+
+# Cert workaround
+cat > 99-lamassu << EOL
+Acquire::https::ubilinux.org::Verify-Peer "false";
+Acquire::https::ubilinux.org::Verify-Host "false";
+EOL
```

Comment:
Hmmm is this necessary? I think we're gonna start using plain Ubuntu and Ubuntu's servers/mirrors on the UP 4000 (as recommended by them).

Reply:
Rafael said this is currently a pending issue and to leave it as is, for now